docs/content/querying/post-aggregations.md

```diff
@@ -54,6 +54,21 @@ The constant post-aggregator always returns the specified value.
 { "type"  : "constant", "name"  : <output_name>, "value" : <numerical_value> }
 ```
 
+### Greatest / Least post-aggregators
```

Comment:
let's add a note to the docs that brings attention to the difference between MAX and GREATEST